integration/use-date-false/google/protobuf/timestamp.ts

```diff
@@ -56,16 +56,7 @@ export const protobufPackage = 'google.protobuf';
  *         .setNanos((int) ((millis % 1000) * 1000000)).build();
  *
  *
- * Example 5: Compute Timestamp from Java `Instant.now()`.
- *
- *     Instant now = Instant.now();
- *
- *     Timestamp timestamp =
- *         Timestamp.newBuilder().setSeconds(now.getEpochSecond())
- *             .setNanos(now.getNano()).build();
- *
- *
- * Example 6: Compute Timestamp from current time in Python.
+ * Example 5: Compute Timestamp from current time in Python.
```

Comment:
I guess this file (and `observable.bin` and `metadata.bin`) changed because I used a different version of `protoc` than when they were compiled. I checked in these changes because after `./update-bins.sh && ./codegen.sh && npm run test -- -u` there are no other unstaged files. It seems that this makes it consistent 😇 

```console
~/git/ts-proto/integration feat/support-date-as-string*
❯ protoc --version
libprotoc 3.13.0
```